lib/Serialization/Serialization.cpp

```diff
@@ -1088,8 +1088,9 @@ void Serializer::writeHeader(const SerializationOptions &options) {
               ++Arg;
               continue;
             }
-          } else if (arg.startswith("-fdebug-prefix-map=")) {
-            // We don't serialize the debug prefix map flags as these
+          } else if (arg.startswith("-fdebug-prefix-map=") ||
+              arg.startswith("-ffile-prefix-map=")) {
```

Comment:
Why don't we check coverage/macro here?

Reply:
@keith Is this an oversight? Should add them here?

Reply:
yes we should. I added the swift flag and it took another month+ to get the clang flag in, so I didn't think to or know to come back and do this. I guess I don't know if there's ever a need to pass that flag through with -Xcc, but it shouldn't be treated differently for sure

Reply:
Done. cc @adrian-prantl in case we need to do anything else (e.g. for lldb)